addons/fcs/functions/fnc_onForceUpdate.sqf

```diff
@@ -1,3 +1,18 @@
+/*
+ * Author: ACE
+ *
+ *
+ * Arguments:
+ * Nothing
```

Comment:
Yeay. Well we can put this as WIP so I can fix this in the same tempo that silentspike updates the script and we just do both at the same time :)

I was just bored and didnt feel like scripting so did this.